### PR TITLE
COS: Add more long-term metrics

### PR DESCRIPTION
**Pull request by cbartz**
### Overview

Add more panels to the dashboard to make it look like this:

Screenshot from 2024-01-19 15-40-32

Also, the CharmHub badge URL is fixed.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated.
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: docs/reference/cos.md [hunk 1]
```diff
@@ -26,7 +26,11 @@ The "GitHub Self-Hosted Runner Metrics" metrics dashboard presents the following
 
 The "GitHub Self-Hosted Runner Metrics (Long-Term)" metrics dashboard displays the following rows:
 
-- General: Displays a panel showing the number of jobs per day.
+- General: Contains the following panels:
+  - Total Jobs
+  - Total unique repositories
+  - Timeseries chart displaying the number of jobs per day
+  - Percentage of jobs with low queue time (less than 60 seconds)
```
Comment (arturo-seijas) [inline docs/reference/cos.md:33]: Can you specify this in the panel title too?

**Review by arturo-seijas — COMMENTED** [review]
Jobs per day might be better visualized as a bar chart instead of a line chart, which seems more suitable for visualizing trends

**Review by gtrkiller — DISMISSED** [review]
LGTM

File: docs/reference/cos.md (hunk 1 above)
Comment (cbartz) [reply docs/reference/cos.md:33]: Just so I understand you correctly, it is already in the panel description, you would prefer it to be in the panel title too?

**Comment by cbartz** [comment]
> Jobs per day might be better visualized as a bar chart instead of a line chart, which seems more suitable for visualizing trends

Like this (thats real data) ?
image

File: docs/reference/cos.md (hunk 1 above)
Comment (arturo-seijas) [reply docs/reference/cos.md:33]: Yes, it will be clearer to an operator what "low" means

**Comment by arturo-seijas** [comment]
> > Jobs per day might be better visualized as a bar chart instead of a line chart, which seems more suitable for visualizing trends
> 
> @arturo-seijas Like this (thats real data) ? image

Yes, that's exactly what I had in mind

File: docs/reference/cos.md (hunk 1 above)
Comment (cbartz) [reply docs/reference/cos.md:33]: Have updated

**Review by arturo-seijas — DISMISSED** [review]

**Review by arturo-seijas — APPROVED** [review]

**Review by jdkandersson — APPROVED** [review]

**Pull request merged**